src/iframe/src/Iframe.js

```diff
@@ -314,21 +319,27 @@ class Iframe extends Component {
     let globals
 
     if (!providedGlobals) {
+      let canvasAPI = contextCanvasAPI
+      if (this.renderer === "framebuffer") {
+        canvasAPI = frameBufferCanvasAPI
+      }
+
       globals = {
         console,
         StateMachine,
         JSON,
         Math,
         Object,
         Array,
-        log: this.logger,
         ...canvasAPI({
+          pixels: this._pixelIntegers,
           ctx: this._canvas.getContext('2d'),
           width: CANVAS_SIZE,
           height: CANVAS_SIZE,
           sprites: this.state.sprites,
           map: this.state.map
         }),
+        log: this.logger,
```

Comment:
Probably best to move this back to where it was originally, less noise

Reply:
Fixed.